dom/attr/attr.js

```diff
@@ -84,22 +84,28 @@ var formElements = {"INPUT": true, "TEXTAREA": true, "SELECT": true},
 			}
 		}
 	},
-	setChildOptions = function(el, value){
-		if(value != null) {
-			var child = el.firstChild,
-				hasSelected = false;
-			while(child) {
-				if(child.nodeName === "OPTION") {
-					if(value === child.value) {
-						hasSelected = child.selected = true;
-						break;
-					}
-				}
-				child = child.nextSibling;
+	_findOptionToSelect = function (parent, value) {
+		var child = parent.firstChild;
+		while (child) {
+			if (child.nodeName === 'OPTION' && value === child.value) {
```

Comment:
I can't remember which version of IE gets `.nodeName` right.  We still support IE9, so maybe double check that you can do this w/o `.toUpperCase()`.  (Or if you see direct `.nodeName` reads everywhere, it's a safe bet that it's ok).